src/Runtime/OMExternalConstant.inc

```diff
@@ -89,7 +93,10 @@ void omMMapBinaryFile(
       filePath = (char *)malloc(filePathLen);
       if (!filePath) {
         fprintf(stderr, "Error while malloc");
-        return;
+#ifdef __MVS__
+        free(tPath);
+#endif
+        return false;
       }
       memcpy(filePath, basePath, baseLen);
```

Comment:
The `memcpy` and adding `\0` can be replaced by a single
```
sprintf(filePath, filePathLen+1, "%s%s%s", basePath, DIR_SEPARATOR, fname);
```

Reply:
Thanks! I updated by using `snprintf`